src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -55,6 +55,20 @@ public ProcessStartInfo(string fileName, string arguments)
             _arguments = arguments;
         }
 
+        /// <devdoc>
+        ///     Specifies the name of the application that is to be started, as well as a set
+        ///     of command line arguments to pass to the application.
+        /// </devdoc>
```

Comment:
`devdoc` is relic of the past (it's not documented anywhere, you had no chance to know about it), please use standard summary tags

```suggestion
        /// <summary>
        /// Specifies the name of the application that is to be started, as well as a set
        /// of command line arguments to pass to the application.
        /// </summary>
```